### ISSUE TITLE
Question about defaultValue configuration option

### ISSUE THREAD

**Issue by mme-private** [ngneat/transloco-keys-manager#8]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[x] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

Tried to change defaultValue configuration option to generate text for non-existing keys, but could't get missing key inside message

## Expected behavior

Some way to reference key value

## Minimal reproduction of the problem with instructions

Changed transloco.config.js file to:

```js
module.exports = {
	rootTranslationsPath: 'src/assets/i18n/',
	langs: ['en', 'lv'],
	keysManager: {
		input: 'src/app',
		translationsPath: 'src/assets/i18n',
		sort: true,
		addMissingKeys: true,
		defaultValue: 'No trans {key}', // tried {{key}}, `No trans ${key}`
		replace: true
	}
};
```
Generated missing key text doesn't contain property names with key, only text like:
```json
{
  "search.title": "No trans {key}"
}
```

## Environment

<pre><code>
Angular version: 8.2.12

For Tooling issues:
- Node version: v12.13.1  
- Platform:  Windows 7 Ultimate 
</code></pre>

**Comment by NetanelBasal** [comment]
We'll add this option.

**Comment by shaharkazaz** [comment]
@mme-private This will be available in v1.2.0 that will be released tomorrow morning 🙂
